core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -47,6 +47,8 @@
 
```

Comment:
Ideally this QueryBuilder is only used for tabs that haven't been refactored to React yet

Reply:
Not really related to this PR, but just FYI and curious how integrated this still is in current architecture

Reply:
QueryBuilder is the entry point to query page. All the requests(index.do) should come here.